runner/app/live/streamer/protocol/trickle.py

```diff
@@ -64,8 +66,10 @@ async def stop(self):
         self.publish_task = None
 
     async def ingress_loop(self, done: asyncio.Event) -> AsyncGenerator[InputFrame, None]:
+        subscribe_queue = self.subscribe_queue
+        publish_queue = self.publish_queue
```

Comment:
What's this change doing? I imagine it was from a previous version where you actually created new queues?

Reply:
Just to be on the self side, I assign them to separate variables in case we change `self.subscribe_queue` somewhere. Since these `self.*` can be accessed from different threads AFAIU